NEWS

```diff
@@ -11,6 +11,20 @@
 ### Features:
 ### Bugfixes:
 
+## 1.14.1 RC1 (May 17, 2023)
```

Comment:
```suggestion
## 1.14.1 (May 17, 2023)
```

Reply:
I think this is not meant to be an RC, or is it?

Reply:
it is RC1
when 1.14.1 is released we remove this